src/App.tsx

```diff
@@ -120,7 +121,7 @@ function ProtectedRoute({
   children,
 }: ProtectedRouteProperties): JSX.Element | null {
   const { pathname } = useLocation();
-  const isProfileFormPath = pathname === '/profile-form';
+  const isProfileFormPath = pathname.includes('/profile');
```

Comment:
With the renaming of the `View user profile` route from `/user-profile` to `/profile/view` on line 208, this check was blocking access to the page.  This improves the specificity so that when a user already has an associated institution, we only skip the the `Complete user profile` forms.

```suggestion
  const isProfileFormPath = pathname.includes('/profile/complete');
```